install/Makefile

```diff
@@ -3,6 +3,13 @@ OS ?= $(shell uname -s | tr '[:upper:]' '[:lower:'])
 INSTALL_PATH ?= /usr/local/bin
 CURL ?= curl --fail -sSL
 TMP ?= /tmp
+EXTENSION=""
+
+ifeq ($(OS),darwin)
```

Comment:
We cannot use a global conditional to map OS to an extension because there's no way to guarantee all projects will use the same convention.

Reply:
Instead, use this pattern:

https://github.com/cloudposse/packages/blob/master/install/Makefile#L128-L142